src/main/java/com/endava/endabank/constants/Strings.java

```diff
@@ -0,0 +1,37 @@
+package com.endava.endabank.constants;
+
+public interface Strings {
+    String ROLE_NOT_FOUND = "The role was not found on the database.";
+    String IDENTIFIER_TYPE_NOT_FOUND = "The identifier type was not found on the database.";
+    String USER_NOT_FOUND = "The user was not found on the database.";
+    String MAIL_SENT = "Mail Sent";
+    String MAIL_FAIL = "There was an error sending the mail.";
```

Comment:
Hi,

This constant has a spelling mistake "MAIL_FAIL" --> "MAIL_FAILED" or --> "MAIL_FAILURE"